core/dbt/include/global_project/macros/get_custom_name/get_custom_alias.sql

```diff
@@ -19,13 +19,17 @@
 
 {% macro default__generate_alias_name(custom_alias_name=none, node=none) -%}
 
-    {%- if custom_alias_name is none -%}
+    {%- if custom_alias_name -%}
```

Comment:
This if/else order makes much more sense to me, and is how it's always been in my head